.github/workflows/CI.yml

```
@@ -0,0 +1,41 @@
+name: CI
+
+on:
+  schedule:
+    - cron: '30 2 * * *'
+  push:
+
+env:
+  IMAGE_NAME: devinsolutions/osticket
+
+jobs:
+  image:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Load image version
+        run: 'echo "IMAGE_VERSION=$(cat image-version)" >> $GITHUB_ENV'
+      - name: Generate image tags
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: ${{ env.IMAGE_NAME }}
+          flavor: |
+            latest=${{ github.ref_name == 'master' }}
+          tags: |
+            type=semver,pattern={{major}},value=${{ env.IMAGE_VERSION }}
+            type=semver,pattern={{major}}.{{minor}},value=${{ env.IMAGE_VERSION }}
+            type=semver,pattern={{version}},value=${{ env.IMAGE_VERSION }}
+      - name: Login to DockerHub
+        if: github.ref_name == 'master'
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Build the image
+        uses: docker/build-push-action@v2
+        with:
+          push: ${{ github.ref_name == 'master' }}
```

```suggestion
          pull: true
          push: ${{ github.ref_name == 'master' }}
```

To make sure it's being built atop of the latest base image.